addon/components/power-select.hbs

```diff
@@ -174,12 +176,15 @@
             @highlightOnHover={{this.highlightOnHover}}
             @groupComponent={{Group}}
             id={{listboxId}}
+            @ariaLabel={{@ariaLabel}}
```

Comment:
passing through accessible labels, listbox should have the labels associated with them too according to the audit